src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryUntypedObject.cs

```diff
@@ -66,11 +71,16 @@ public static FormulaValue Value_UO(IRContext irContext, UntypedObjectValue[] ar
 
                 return new NumberValue(irContext, number);
             }
+            else if (impl.Type == FormulaType.Boolean)
```

Comment:
>impl.Type == FormulaType.Boolean [](http://example.com/codeflow?start=21&length=32)

Calling `Value(false)` today is a compiler error. If we allow this, we should also let the "normal" Value function accept boolean inputs as well.

Reply:
Done